packages/dart/npt_flutter/lib/features/settings/view/settings_view.dart

```diff
@@ -43,13 +43,13 @@ class SettingsView extends StatelessWidget {
                         crossAxisAlignment: CrossAxisAlignment.start,
                         mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                         children: [
-                          // gapH30,
-                          CustomTextButton.discord(),
-                          CustomTextButton.email(),
+                          gapH10,
+                          CustomTextButton.backUpYourKey(),
                           CustomTextButton.faq(),
-                          CustomTextButton.privacyPolicy(),
+                          CustomTextButton.email(),
+                          CustomTextButton.discord(),
                           CustomTextButton.feedback(),
-                          CustomTextButton.backUpYourKey(),
+                          CustomTextButton.privacyPolicy(),
```

Comment:
Don't rearrange this unnecessarily, it makes the videos and screenshots on the docs site out of date.

Reply:
@tinashe404 asked me to rearrange it since this order makes more sense logically. We'll stick with this arrangement going forward.